analyzers/tests/SonarAnalyzer.UnitTest/TestCases/JwtSigned.cs

```diff
@@ -22,6 +22,7 @@ void DecodingWithDecoder(JwtDecoder decoder)
             string decoded;
             decoded = decoder.Decode(invalidToken, secret, true);
             decoded = decoder.Decode(invalidToken, secret, false); // Noncompliant {{Use only strong cipher algorithms when verifying the signature of this JWT.}}
+            decoded = decoder?.Decode(invalidToken, secret, false); // Noncompliant
```

Comment:
Same comment also here.

Reply:
Same as above (just to have the same thing covered in two rules in case we delete one of them in the far future)